readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -77,7 +77,13 @@ function attach_elastic_search_query(data) {
                         }
 
                         // Creating the result from elements
-                        var link = doc.link + DOCUMENTATION_OPTIONS.FILE_SUFFIX;
+                        var suffix = DOCUMENTATION_OPTIONS.FILE_SUFFIX;
+                        // Since sphinx 2.2.1 FILE_SUFFIX is .html for all builders,
+                        // and there is a new BUILDER option.
+                        if ('BUILDER' in DOCUMENTATION_OPTIONS && DOCUMENTATION_OPTIONS.BUILDER === 'readthedocsdirhtml') {
```

Comment:
Isn't it better to use `.endswith('dirhtml')` here? That way, we allow people using a different builder. See https://docs.readthedocs.io/en/stable/builds.html#writing-your-own-builder

Reply:
That case will be only valid for a custom installation with their own builders, not sure if users have that level of customization (there are a lot of parts to change the code to make that work)